docs/restate/cli.mdx

```diff
@@ -15,10 +15,10 @@ You can use the CLI to interact with Restate, and manage your services, deployme
 Install the Restate CLI via:
 
 <Tabs groupId="operating-systems">
-<TabItem value="lin" label="npm">
+<TabItem value="lin" label="npx">
 
 ```shell
-npm i @restatedev/restate
+npx @restatedev/restate
```

Comment:
Maybe for a tool like the CLI a global install with npm would be better? Then you can use it as `restate` afterwards? With npx you don't really install it.
```suggestion
npm i -g @restatedev/restate
```

Reply:
Maybe I should update the `Get Restate` page as well then

Reply:
I think this would also work (at least I tried it locally and it worked on my machine). In any case we should settle for the same thing.

Reply:
Btw: Currently, we use npx for the server and npm install for the cli on https://restate.dev/get-restate/. That's also something we should pull straight.